modules/raft/src/integrationTest/java/org/apache/ignite/internal/raft/ItLearnersTest.java

```diff
@@ -73,19 +77,38 @@
  */
 @ExtendWith(ConfigurationExtension.class)
 public class ItLearnersTest extends IgniteAbstractTest {
-    private static final String RAFT_GROUP_NAME = "test";
+    private static final ReplicationGroupId RAFT_GROUP_ID = new ReplicationGroupId() {
+        @Override
+        public String toString() {
+            return "test";
```

Comment:
Shouldn't there be a dedicated method for the id? `toString()` usually has a different meaning

Reply:
Again, I don't understand. This class wasn't introduced by me, I'm just playing by the rules introduced by others

Reply:
Didn't noticed that at first